lib/msf/core/payload_generator.rb

```diff
@@ -255,7 +255,7 @@ def multiple_encode_payload(shellcode)
           cli_print "#{encoder_opt[0]} not found continuing..."
           next
         end
-        encoder_mod.datastore.import_options_from_hash(datastore)
+        encoder_mod.datastore.merge!(datastore)
```

Comment:
With the feature flag added does this need to be update? 
```suggestion
        if encoder_mod.datastore.is_a?(Msf::DataStoreWithFallbacks)
          encoder_mod.datastore.merge!(datastore)
        else
          encoder_mod.datastore.import_options_from_hash(datastore)
        end
```